src/test/java/io/github/fileanalysissuite/adaptersdk/impls/jaxrs/AdapterSDKMockTest.java

```diff
@@ -70,4 +74,24 @@ void testAdapterDescriptorGet()
 
         assertThat(actualDescriptor, equalTo(expectedDescriptor));
     }
+
+    @Test
+    void testItemMetadataDateFormat()
```

Comment:
This test doesn't belong here (i.e. in `AdapterSDKMockTest`), because it doesn't use the mock adapter in any way.  It is actually directly unit testing the `InstantFunctions` method.  Either create a new `InstantFunctionsTest.java` and move this test into it, or else just get rid of it altogether.  It's not really needed because you're already testing the new functionality with your changes to the `AdapterSDKFakeTest`.

Reply:
I see what you mean, I have now removed this test